src/commands/mod.rs

```diff
@@ -32,6 +33,8 @@ pub enum Command {
     QRCode(qrcode::QRCodeOpts),
     /// Sends signed messages to the Internet computer.
     Send(send::SendOpts),
+    /// Make a proposal to upgrade an SNS-controlled canister.
```

Comment:
I've been comments for the opts and this enum consistently.
```suggestion
    /// Signs a ManageNeuron message to submit a UpgradeSnsControlledCanister
    /// proposal.
```

Reply:
Done.